experienceStyles.css

```diff
@@ -0,0 +1,310 @@
+#experience {
```

Comment:
similar comment as the scripts file - consider having a shared stylesheet to condense any repeated styles across your different pages, and then having the page-specific stylesheets